nengo/builder/transforms.py

```diff
@@ -6,7 +6,13 @@
 from nengo.builder.signal import Signal
 from nengo.exceptions import BuildError
 from nengo.rc import rc
-from nengo.transforms import Convolution, Dense, NoTransform, Sparse
+from nengo.transforms import (
+    Convolution,
+    ConvolutionTranspose,
```

Comment:
Seeing this made me think, maybe we should add `Conv`/`ConvTranspose` aliases for these?

Reply:
It would certainly look more familiar for those coming from many other places, like TensorFlow/Keras. I would be fine with adding those.